src/App.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable react/react-in-jsx-scope */
+/* eslint-disable react/jsx-filename-extension */
```

Comment:
- [ ] Let's remove these `eslint-disable` lines here. After using the [eslint config provided by Microverse](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json), you will not have any linter errors here.